lib/collection/src/collection_manager/holders/proxy_segment.rs

```diff
@@ -158,6 +158,57 @@ impl ProxySegment {
             }
         }
     }
+
+    /// Propagate changes in this proxy to the wrapped segment
+    ///
+    /// This propagates:
+    /// - delete (or moved) points
+    /// - deleted payload indexes
+    /// - created payload indexes
+    ///
+    /// This is required if making both the wrapped segment and the writable segment available in a
+    /// shard holder at the same time. If the wrapped segment is thrown away, then this is not
+    /// required.
+    pub(super) fn propagate_to_wrapped(&self) -> OperationResult<()> {
+        let wrapped_segment = self.wrapped_segment.get();
+        let mut wrapped_segment = wrapped_segment.write();
+        let op_num = wrapped_segment.version() + 1;
```

Comment:
same, let's just use `wrapped_segment.version()`, it should be enough to apply operation and it is used like this in optimizer